.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,16 +1,18 @@
 ---
-name: Bug Report
-about: Create a report to help us improve
+name: Bug Report(.Net SDK)
+about: Create a report to help us improve AI .NET SDK
 labels: bug
 ---
 
-**Describe your environment.** Describe any aspect of your environment relevant to the problem:
+**Is this a bug in .NET SDK?**
+             If yes, proceed to Bug Template. 
+             If not, please file it under the appropriate AI SDK repro path: [Javascript SDK](https://github.com/Microsoft/ApplicationInsights-js/issues) ; [Java SDK](https://github.com/Microsoft/ApplicationInsights-Java/issues) ; [Node.JS SDK](https://github.com/Microsoft/ApplicationInsights-node.js/issues)
+
+
+-----------------Bug Template--------------------------------------------------------------
+
 * SDK version: 
 * .NET runtime version (.NET or .NET Core, TargetFramework in the .csproj file): 
```

Comment:
If hosting information and OS are relevant to the issue (as they are in many cases), where does they get mentioned?